client/src/pages/Home/Home.js

```diff
@@ -11,6 +11,11 @@ import { Modal, Button, Input, Row } from "react-materialize";
 import Nav from '../../components/Nav';
 import "./Home.css";
 import API from "../../utils/API";
+import axios from 'axios';
```

Comment:
Hi Jacob, this code is incorrect based on our architecture. the API call needs to be initiated here via a method to call the API file in the utils folder, there the axios call is made from. I cannot approve this until that is done. The method is similar to what was done in the NYT react homework (which using an external API keys there are extra steps that should be done to hide it in a .env file and using the routes/controller to make the actual call to smarty streets, but that can be a next step after getting the front end code in the correct locations). If you have quesitons please slack me! thanks1